freeipa/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClientBuilder.java

```diff
@@ -193,21 +187,14 @@ private String connect(String user, String pass, String apiAddress, int port)
                 throw new FreeIpaClientException(String.format("Encountered unexpected response from "
                         + "FreeIPA; details:%n%n"
                         + "code: %s%n"
-                        + "headers: %s", response.getStatusLine().getStatusCode(), response.getAllHeaders()));
+                        + "headers: %s", response.getStatusLine().getStatusCode(), response.getAllHeaders()),
+                        response.getStatusLine().getStatusCode());
```

Comment:
I don't think this `response.getStatusLine().getStatusCode()` is necessary

Reply:
It's necessary for the calling function to determine if the exception was due to not being authorized.  We want to retry logging it to different IPA servers for failover, unless the exception is because we're unauthorized.

Reply:
ok, I missed the parenthesis